src/Octoshift/Services/VersionChecker.cs

```diff
@@ -26,7 +26,7 @@ public async Task<bool> IsLatest()
 
         curVersion = curVersion[..latestVersion.Length];
 
-        return curVersion == latestVersion;
+        return latestVersion.CompareTo(curVersion) <= 0;
```

Comment:
I do wonder if we should use a semantic versioning library here to properly compare the versions - that would be the safest option - but in practice, I don't think it's necessary.